_posts/en/newsletters/2021-12-15-newsletter.md

```diff
@@ -0,0 +1,117 @@
+---
+title: 'Bitcoin Optech Newsletter #179'
+permalink: /en/newsletters/2021/12/15/
+name: 2021-12-15-newsletter
+slug: 2021-12-15-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a proposal to allow relay of
+transactions with zero-value outputs in some cases and summarizes a
+discussion about preparing LN for the adoption of PTLCs.  FIXME:harding
+also included are...
+
+## News
+
+- **Adding a special exception for certain uneconomical outputs:**
+  since our description in [Newsletter #162][news162 unec], Jeremy Rubin
+  has [renewed][rubin unec] discussion on the Bitcoin-Dev mailing list
+  about allowing transactions to create outputs with values below the
+  [dust limit][topic uneconomical outputs].  The dust limit is a
+  transaction relay policy that nodes use to discourage users from
+  creating UTXOs that don't make economic sense to spend.  UTXOs need to
+  be stored by at least some full nodes until spent, and in some cases
+  be able to be retrieved quickly, so allowing *uneconomical outputs*
+  may create problems for no good reason.
+
+    However, there may be a use for zero-value outputs in [CPFP][topic
+    cpfp] fee bumping where the none of the funds from the transaction
+    being fee bumped can be spent---all the funds used for fee bumping
+    need to come from separate UTXOs, such as in [eltoo][topic eltoo].
```

Comment:
aren't anchor outputs the more relevant example here? Anchor outputs are already in practice while eltoo is far from being implemented.

Reply:
I don't think there's any advantage to anchor outputs being zero-value.  The transaction containing them *needs* to be fee-bumped, so they might as well contain at least the dust amount of funds to contribute towards that fee bumping.

By comparison, most transactions in an eltoo or statechain system will remain offchain (even if they can all hypothetically be put onchain) and so they can't each pay some value towards an output without quickly draining the system of value.  This is usually solved by them using some version of `SIGHASH_SINGLE` to allow adding additional outputs, but Jeremy Rubin made a case that avoiding SINGLE and using CPFP instead might avoid RBF pinning attacks.